Source/WebCore/platform/graphics/GlyphPage.h

```diff
@@ -135,7 +136,7 @@ class GlyphPage : public RefCounted<GlyphPage> {
         ++s_count;
     }
 
-    const Font& m_font;
+    CheckedRef<const Font> m_font;
```

Comment:
But CheckedRef here. That seems like arbitrary. We should probably state the reasoning in the commit message.